service/worker/migration/activities.go

```diff
@@ -346,3 +351,13 @@ func (a *activities) GenerateReplicationTasks(ctx context.Context, request *gene
 
 	return nil
 }
+
+func allNamespaceDivisions(query string) string {
+	// This should be a value that is never used as a namespace division.
+	const matchAll = searchattribute.TemporalNamespaceDivision + ` != "__never_used__"`
+
+	if strings.TrimSpace(query) == "" {
+		return matchAll
+	}
+	return fmt.Sprintf(`(%s) AND (%s)`, query, matchAll)
```

Comment:
Do we need to check if query already contains filter on division? Or will this become repeated if called multiple times? Also, namespace delete has the same problem of this.

Reply:
the query is written so that it doesn't affect the result. so if the original is `status = running`, that would normally get modified to `(status = "running") and (nsdiv = "")` and match only empty string for nsdiv. if passed through this function, then it becomes `(status = "running") and (nsdiv != "never_used")`, and that gets used without further modification, so it matches running wfs with any value for nsdiv (except for "never_used" which should never be used).

but then if the query starts as `nsdiv = "scheduler"`, it gets changed to `(nsdiv = "scheduler") and (nsdiv != "never_used")`, which will not be modified further, and which still matches only nsdiv = "scheduler".

repeated calls will just add more always-true AND clauses, which is harmless, though we should avoid that.

Reply:
I changed namespace delete activities too now